packages/editor/src/components/widgets/tag/Tags.tsx

```diff
@@ -1,80 +1,165 @@
 import './Tags.css';
-import '../popover/Popover.css';
-import * as Popover from '@radix-ui/react-popover';
-import { memo, useState } from 'react';
-import { useKeyboard } from 'react-aria';
-import { useEditorContext } from '../../../context';
+import { memo, useEffect, useRef, useState } from 'react';
+import { useEditorContext, useMeta } from '../../../context';
 import IvyIcon from '../IvyIcon';
 import { IvyIcons } from '@axonivy/editor-icons';
-import { Fieldset, useFieldset } from '../fieldset';
-import { Input } from '../input';
+import { useCombobox } from 'downshift';
+import { useKeyboard } from 'react-aria';
+import { MessageText } from '../message/Message';
 
 const Tags = (props: { tags: string[]; onChange: (tags: string[]) => void }) => {
-  const [newTag, setNewTag] = useState('');
-  const [isOpen, setOpen] = useState(false);
+  const { elementContext } = useEditorContext();
+
+  const { data } = useMeta('meta/workflow/tags', elementContext, []);
+  const [dropDownTags, setDropDownTags] = useState(data.filter(tag => !props.tags.includes(tag)));
+  const [addValue, setAddValue] = useState('Add');
```

Comment:
In my current solution, addValue is used as the general input value. I need to initialize useState('Add') at the beginning so that the word "Add" is displayed on the Add button. Leaving this empty would only show the plus sign. Upon clicking the button, the "Add" text should disappear and the input field should be empty so that the user can start the tag from scratch without having to delete the word "Add" first. Once the tag is added, I reset this value to "Add" in the addTag method so that the Add button displays the text "Add" instead of just the plus icon again.

Reply:
Ahhh, didn't get that. Then it is ok 👍